packages/react/__tests__/__snapshots__/PublicAPI-test.js.snap

```diff
@@ -5299,262 +5299,6 @@ Map {
   },
   "MultiSelect" => Object {
     "$$typeof": Symbol(react.forward_ref),
-    "Filterable": Object {
```

Comment:
I just want to call out that this isn't a breaking change because it was deprecated all the way back in v10.41.0, https://github.com/carbon-design-system/carbon/pull/9008 and stated that it would be removed in v11. I don't know of a reason why this was left in, we probably just missed it in the lead-up to v11.